internal/internal_task_handlers.go

```diff
@@ -1754,7 +1754,7 @@ func (ath *activityTaskHandlerImpl) Execute(taskQueue string, t *workflowservice
 
 	workflowType := t.WorkflowType.GetName()
 	activityType := t.ActivityType.GetName()
-	activityMetricsScope := metrics.GetMetricsScopeForActivity(ath.metricsScope, workflowType, activityType)
+	activityMetricsScope := metrics.GetMetricsScopeForRPC(ath.metricsScope, workflowType, activityType, taskQueue)
```

Comment:
I think we should just update `GetMetricsScopeForActivity` to take additional parameter like in #452

Reply:
Well shoot. I would have liked to make those adjustments so I could have contributed here, but I guess we can just close this PR since you've done the work already. Thanks for resolving the issue.

Reply:
Sorry, I've done before I've seen your PR, you can make adjustments here, no big deal. Let me cancel mine instead.

Reply:
That's kind of you @vitarb. Thank you! I have so enjoyed working with temporal over the last several months and appreciate all the amazing work you have all done.

Reply:
Thanks for your contribution to our community!